### ISSUE TITLE
[Core] Stuck in combat (combat bug) in Stromgarde Keep, Arathi Highlands

### ISSUE THREAD

**Issue by Sinyr** [Atlantiss/NetherwingBugtracker#1542]
[//]: # (Enclose links to things related to the bug using http://wowhead.com or any other TBC database.)
[//]: # (You can use screenshot ingame to visual the issue.)
[//]: # (Write your tickets according to the format:)
[//]: # ([Quest][Azuremyst Isle] Red Snapper - Very Tasty!)
[//]: # ([NPC] Magistrix Erona)
[//]: # ([Spell][Mage] Fireball)
[//]: # ([Npc][Drop] Ghostclaw Lynx)
[//]: # ([Web] Armory doesnt work)

**Current behaviour**:
I got stuck in combat around the place in the screenshot, inside Stromgarde Keep in Arathi Highlands. There were a couple of monsters at that place, but I killed all of them all the way from the entrance, including the bosses in the main room, but I was still in combat. Finally got out of combat outside the keep, when I got near the ogres in the back.

![screenshot_37](https://user-images.githubusercontent.com/9539181/46893233-94a06a00-ce70-11e8-899c-9b2142a712b0.png)

(Also .gps incorrectly says I'm outside, not sure if it matters)

**Expected behaviour**: 
Should leave combat if no nearby monsters are alive or chasing you.

**Server Revision**: 
2161

**Comment by Easyfraggy** [comment]
We need recreation steps, if you can't recreate the situation it's near impossible for us to do so and try to fix this issue.

**Comment by wtRCH** [comment]
Recreation steps:
- Kill everything inside the big keep in the middle of Stormgarde Keep
- Wait

**Comment by Easyfraggy** [comment]
So i cleared the keep and waited, no combat bug.....

**Comment by Easyfraggy** [comment]
It's not location based it happens due to some spell from player or mob or smth else.

**Comment by Arnamik12** [comment]
I had an issue like this in deadmines, was a rat hit by a non damage spell.

**Comment by Sinyr** [comment]
As a Paladin, the only spell that could've somehow hit something else than my main target was Consecration. I think I did use it around this spot, since we got attacked by 3-4 NPCs there. It could've been a critter bugging out or something, but I don't recall seeing one there. My leveling buddy (Mage) also used some form of AoE there, but he wasn't stuck in combat like me, even though we were right next to each other.
